drivers/eeprom/eeprom_at2x.c

```diff
@@ -606,7 +590,7 @@ static const struct eeprom_driver_api eeprom_at2x_api = {
 #ifndef DT_INST_0_ATMEL_AT24_WP_GPIOS_CONTROLLER
 #define DT_INST_0_ATMEL_AT24_WP_GPIOS_CONTROLLER NULL
 #define DT_INST_0_ATMEL_AT24_WP_GPIOS_PIN 0
-#define DT_INST_0_ATMEL_AT24_WP_GPIOS_FLAGS 0
+#define DT_INST_0_ATMEL_AT24_WP_GPIOS_FLAGS GPIO_ACTIVE_LOW
```

Comment:
These are just default values in case no WP pin is set in DTS. They can remain as `0` or `GPIO_ACTIVE_LOW`. Their value is not used. It is just to satisfy the C macro for generating the device instance.